examples/correlation_alignment_example.py

```diff
@@ -66,21 +66,10 @@
         ax_list[0].set_title("Raw Data")
         data = data["t2":f_range]
         data.ift("t2")
-        rough_center = (
-            abs(select_pathway(data, signal_pathway))
-            .C.convolve("t2", 0.3e-3)
-            .mean_all_but("t2")
-            .argmax("t2")
-            .item()
-        )
-        logger.info(strm("Rough center is:", rough_center))
-        data.setaxis("t2", lambda x: x - rough_center).register_axis({"t2": 0})
-        data.ft("t2")
-        data.ift("t2")
```

Comment:
this makes it not work.  While it shouldn't *help*, it also shouldn't *break* the code.  I think that the way the code is written, it doesn't deal well if it starts at a value that's not near 0?

Reply:
fixed below

Reply:
I don't see how this pertains to the updated hermitian function

Reply:
you are always supposed to make sure you can rebuild the sphinx gallery,  and it had broken this.